Source/Core/FeatureDetection.js

```diff
@@ -269,6 +267,10 @@ define([
         }
     }
 
+    function supportsCreateImageBitmap() {
```

Comment:
I don't think this or `supportsImageBitmapOptionsSync` is really needed you can delete them and simplify the code.  You should just be able to use `Resource.supportsImageBitmapOptions` everywhere we need to know, I did a quick look into this locally and everywhere this would be used is already promise-based.